src/components/button-bar/button-bar.component.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+import propTypes from "@styled-system/prop-types";
+import StyledButtonBar from "./button-bar.style";
+import Button from "../button";
+import IconButton from "../icon-button";
+
+const ButtonBar = ({ children, size, iconPosition, buttonType, ...rest }) => {
+  const getBtnProps = (child) => {
+    const btnProps = {
+      ...child.props,
+      buttonType: "secondary",
+      size,
+      iconPosition,
+      fullWidth: false,
+      "aria-label":
+        child.type === IconButton ? child.props?.children?.props?.type : "",
```

Comment:
Apologies I missed this during the first review, `IconButton` also supports `ariaLabel` so I think we should only override it if it isn't set here.

```
"aria-label":
        child.type === IconButton ? child.props?.ariaLabel || child.props?.children?.props?.type : ""
```

Reply:
Done